openpype/plugins/inventory/remove_and_load.py

```diff
@@ -0,0 +1,50 @@
+from openpype.pipeline import InventoryAction
+from openpype.pipeline import get_current_project_name
+from openpype.pipeline.load.plugins import discover_loader_plugins
+from openpype.pipeline.load.utils import (
+    get_loader_identifier,
+    remove_container,
+    load_container,
+)
+from openpype.client import get_representation_by_id
+
+
+class RemoveAndLoad(InventoryAction):
+    """Delete inventory item and reload it."""
+
+    label = "Remove and load"
+    icon = "refresh"
+
+    def process(self, containers):
+        project_name = get_current_project_name()
+        for container in containers:
+            # Get loader
+            loader_name = container["loader"]
+            loaders = discover_loader_plugins(project_name=project_name)
```

Comment:
```suggestion
        project_name = get_current_project_name()
        loaders = discover_loader_plugins(project_name=project_name)
        for container in containers:
            # Get loader
            loader_name = container["loader"]
```

Reply:
I'd even take this a step further and precache the by loader name identifier too

```suggestion
        project_name = get_current_project_name()
        loaders = discover_loader_plugins(project_name=project_name)
        loaders_by_name = {get_loader_identifier(plugin) for plugin in loaders}
        for container in containers:
            # Get loader
            loader_name = container["loader"]
            loader = loaders_by_name.get(loader_name, None)
            if not loader:
                raise RuntimeError(
                    "Failed to get loader '{}', can't remove "
                    "and load container".format(loader_name)
                )
```

Also added the loader name that was not found in the error message for clarity.

Reply:
I'd even take this a step further and precache the by loader name identifier too

```suggestion
        project_name = get_current_project_name()
        loaders = discover_loader_plugins(project_name=project_name)
        loaders_by_name = {get_loader_identifier(plugin) for plugin in loaders}
        for container in containers:
            # Get loader
            loader_name = container["loader"]
            loader = loaders_by_name.get(loader_name, None)
            if not loader:
                raise RuntimeError(
                    "Failed to get loader '{}', can't remove "
                    "and load container".format(loader_name)
                )
```

Also added the loader name that was not found in the error message for clarity.

Reply:
@BigRoy I don't think your suggestion would work because we would only get a loader indentifier that way, but not the loader itself. `set` doesn't have a `get` method either but that's no big deal.

Reply:
Sorry, @Sharkitty - typo. Fixed my suggestion. It was supposed to be a dict.

Reply:
Ah! Makes a lot more sense now ^^ Thanks you